<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerManagementSystemClass</class>
 <widget class="QMainWindow" name="PlayerManagementSystemClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayerManagementSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>537</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="create_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Create a new management system</string>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="continue_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Continue with a old management system</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>EXIT</string>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="group_members">
         <property name="text">
          <string>Group Members </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ID_242">
         <property name="text">
          <string>190041242 Idrissa Mahamoudou Dicko</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ID_245">
         <property name="text">
          <string>190041245 Gandega Abubakar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ID_246">
         <property name="text">
          <string>190041246 Shalanyuy Nabil Kindzeka</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ID_247">
         <property name="text">
          <string>190041247 Mbouwap Njoya Zouleiha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="course_teachers">
         <property name="text">
          <string>Course Teacher Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="teachers">
         <property name="text">
          <string> Faisal Hussain, Tanjila Alam Sathi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>250</y>
      <width>176</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PlayerManagementSystem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit_btn</sender>
   <signal>clicked()</signal>
   <receiver>PlayerManagementSystemClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
